pytorch_lightning/trainer/properties.py

```diff
@@ -185,7 +185,21 @@ def progress_bar_dict(self) -> dict:
         """ Read-only for progress bar metrics. """
         ref_model = self.model if not self.data_parallel else self.model.module
```

Comment:
```suggestion
        ref_model = self.get_model()
```
so for ddp ref_model is just model and not model.module???

Reply:
no but there is some shenanigans going on in the trainer that will make it so that self.model is the unwrapped model. 
This whole model retrieval will be addressed and made more transparent by our refactoring in #4510